libs/components/http/src/lib/modules/auth-http-client/auth-interceptor.spec.ts

```diff
@@ -297,4 +322,30 @@ describe('Auth interceptor', () => {
       jasmine.objectContaining(expectedTokenArgs)
     );
   });
+
+  // describe('with missing SkyAppConfig', () => {
```

Comment:
Eh?

Reply:
Sigh - good catch. That was code that I temporarily brought over from a different spec as a template as I was working on the code above and I didn't intend to save and commit. I have removed it.